docs/discovery-naming-conventions.md

```diff
@@ -1,23 +1,22 @@
-# Discovery Naming Conventions
+# Discovery naming conventions
 
-In order to load balance and route traffic to backend systems, Gimbal must
-discover the backends and sync them to the Gimbal cluster. This is done
-by the Gimbal discovery components, such as the Kubernetes discoverer and the
-OpenStack discoverer.
+To load balance and route traffic to backend systems, Gimbal must
+discover the backends and synchronize them to the Gimbal cluster. This is done
+by the Gimbal discovery components -- currently the Kubernetes Discoverer and the
+OpenStack Discoverer.
 
 During the discovery process, Gimbal translates the discovered backends into
-Kubernetes Services and Endpoints. The name of the discovered Services and
```

Comment:
The "Discovered Name" is a concept that gets mentioned a couple of more times in this document. Not sure if we should maintain the definition of "Discovered Name" here?